### DISCUSSION TITLE
How to build next.js from source (github)

### DISCUSSION THREAD

**Discussion by tienlx97** [vercel/next.js#55867] [Help]
### Summary

How can I build Next.js to package after cloning this open-source project?

Why must I build Next.js instead of fetching it from npmjs?

- I want to use experimental versions of React and React DOM. I enabled some flags in ReactFeaturedFlag.js such as: unstable_createEventHandler, unstable_legacyHidden, unstable_scope, jsx, jsxs,... 
- To do this, I replaced Next.js's React and React DOM in packages/next/src/compiled with my own versions. 
- Therefore, I must rebuild the Next.js package. I run **pnpm build** then copy packages/next to my project (using turborepo). But seem not work, I got some error: mising @next/swc (not permission), require(next/dist/...) not found
- I can link Nextjs by using local link
**"next":"link:.../packages/next"**. But I do not like this way

Can you help me please 🥺

**Comment by jimmy-ventura-elasticpath** [comment]
Additionally building without playwright would be very useful
